base_station/gui/src/static/rover_msgs.json

```diff
@@ -913,6 +919,7 @@
         "TestEnable",
         "Waypoint",
         "WheelSpeeds",
-        "Xbox"
+        "Xbox",
+        "ZedGimbalPosition"
```

Comment:
ZedGimbalData

Reply:
This one doesn't make sense to change since it would be confusing as it would always match one channel name and not the other.